src/sentry/scim/endpoints/members.py

```diff
@@ -274,10 +273,18 @@ def patch(self, request: Request, organization, member):
             # we only support setting active to False which deletes the orgmember
             if self._should_delete_member(operation):
                 self._delete_member(request, organization, member)
+                metrics.incr(
+                    "sentry.scim.member.update",
+                    sample_rate=1.0,
+                    tags={"organization": organization},
+                )
                 return Response(status=204)
             else:
                 raise SCIMApiError(detail=SCIM_400_INVALID_PATCH)
 
+        metrics.incr(
+            "sentry.scim.member.update", sample_rate=1.0, tags={"organization": organization}
+        )
```

Comment:
The current action of this endpoint is limited to only setting a member's `active` attribute to False which deletes the orgmember. All the code below line 283 will never run, but we include a second metric incr in case future updates allow more actions

Reply:
you can also call it delete so the person who's editing the endpoint in the future has no option but creating a new metric 😛

Reply:
ok, will change to delete and get rid of the second unused metric